packages/dropdowns.next/src/types/index.ts

```diff
@@ -252,3 +226,98 @@ export interface ITagGroupProps {
   /** Provides the current selection */
   selection: IOption[];
 }
+
+export interface IMenuListProps extends HTMLAttributes<HTMLUListElement> {
+  /** Appends the menu to the element provided */
+  appendToNode?: Element | DocumentFragment;
+  /** Attaches an arrow that points towards the menu trigger */
+  hasArrow?: boolean;
+  /** Applies compact styling */
+  isCompact?: boolean;
+  /** Determines menu expansion */
+  isExpanded?: boolean;
+  /** Provides a list of acceptable fallback placements */
+  fallbackPlacements?: GardenPlacement[];
+  /** Sets the `max-height` of the menu */
+  maxHeight?: IListboxProps['maxHeight'];
+  /** Sets the `min-height` of the menu */
+  minHeight?: IListboxProps['minHeight'];
+  /** Adjusts the placement of the menu */
+  placement?: GardenPlacement;
+  /** Provides ref access to the associated trigger element */
+  triggerRef: RefObject<HTMLElement>;
+  /** Sets the `z-index` of the menu */
+  zIndex?: IListboxProps['zIndex'];
+}
+
+export interface IMenuButtonProps extends IButtonProps {
+  ref: RefObject<HTMLButtonElement>;
+}
```

Comment:
Kind of realizing I'm not sure the best way to write `jsdoc` for string or callback. Do you have any suggestions? I'm thinking it would probably be better to inline that type right on the `button` prop instead of exporting a totally separate interface.